pouta_blueprints/views/users.py

```diff
@@ -202,3 +210,27 @@ def put(self, user_id):
             user.is_blocked = False
         db.session.add(user)
         db.session.commit()
+
+
+class UserGroupOwner(restful.Resource):
+    parser = reqparse.RequestParser()
+    parser.add_argument('make_group_owner', type=bool, required=True)
+
+    @auth.login_required
+    @requires_admin
+    def put(self, user_id):
+        args = self.parser.parse_args()
+        make_group_owner = args.make_group_owner
+
+        user = User.query.filter_by(id=user_id).first()
```

Comment:
Doesn't flask also have a get() for these occasions? Or do you just want to avoid catching the error a get would throw if the key is not found?